boards/arm/nucleo_h745zi_q/nucleo_h745zi_q.dtsi

```diff
@@ -39,3 +39,19 @@
 	d2ppre2 = <2>;
 	d3ppre = <2>;
 };
+
+&fdcan1 {
+	pinctrl-0 = <&fdcan1_rx_pb8 &fdcan1_tx_pb9>;
+	pinctrl-names = "default";
+	bus-speed = <125000>;
+	bus-speed-data = <2000000>;
+	status = "disabled";
+};
+
+&fdcan2 {
+	pinctrl-0 = <&fdcan2_rx_pb12 &fdcan2_tx_pb13>;
+	pinctrl-names = "default";
+	bus-speed = <125000>;
+	bus-speed-data = <2000000>;
+	status = "disabled";
+};
```

Comment:
Similarly to other peripheral configurations, this should rather be defined in `nucleo_h745zi_q_m7.dts`

Reply:
> Similarly to other peripheral configurations, this should rather be defined in `nucleo_h745zi_q_m7.dts`

H745 ist special, since it has two cores. Moving it to _m7 will mean, same way it must be added to _m4 in case m4 shall use can.
Current implementation has the advantage, that fdcan can be used by both m4 or m7 cores just by adding zephyr,can-bus=fdcanX entry.

Reply:
> H745 ist special, since it has two cores. Moving it to _m7 will mean, same way it must be added to _m4 in case m4 shall use can.

This would also apply to all other peripherals. My request is for consistency sake. Similar approach is (or should be) already in use for other h7 dual core targets.

Reply:
> Similarly to other peripheral configurations, this should rather be defined in `nucleo_h745zi_q_m7.dts`

Moved to mentionde dts file.